src/common/QLogging.hpp

```diff
@@ -31,3 +31,4 @@ Q_DECLARE_LOGGING_CATEGORY(chatterinoWebsocket);
 Q_DECLARE_LOGGING_CATEGORY(chatterinoWidget);
 Q_DECLARE_LOGGING_CATEGORY(chatterinoWindowmanager);
 Q_DECLARE_LOGGING_CATEGORY(chatterinoHighlights);
+Q_DECLARE_LOGGING_CATEGORY(chatterinoRecentMessages);
```

Comment:
Originally, logging categories were kept in alphabetic order. @pajlada (supposedly unintentionally?) broke with this tradition in 7ccf60111d5e8ba54c070fe09d926bdff5290da5 but I would suggest we keep it up for new additions.